lib/profilefunctions.js

```diff
@@ -163,7 +163,7 @@ function init (profileData) {
   profile.getUnits = function getUnits (spec_profile) {
     var pu = profile.getCurrentProfile(null, spec_profile)['units'] + ' ';
     if (pu.toLowerCase().includes('mmol')) return 'mmol';
-    return 'mgdl';
+    return 'mg/dl';
```

Comment:
This might have consequences for systems using Nightscout data, but let's change it anyway

Reply:
@sulkaharo I only found few places in NS using this function and in each one of those same issue occurred, and changing it to mg/dl fixes it. So I don't think there should be any consequences apart from fixing those issues